app/src/main/java/com/github/gotify/service/WebSocketConnection.java

```diff
@@ -104,7 +123,20 @@ public synchronized void close() {
         }
     }
 
+    public synchronized void scheduleReconnect(long delay, int reason) {
```

Comment:
Please specify in what the unit of "delay" is. Or even better let the caller pass the unit
```suggestion
    public synchronized void scheduleReconnect(long delay, TimeUnit unit) {
```